compiler/cbmc/src/utils.rs

```diff
@@ -42,3 +44,56 @@ macro_rules! btree_string_map {
         (BTreeMap::from_iter(vec![$($x),*].into_iter().map(|(k,v)|(k.to_string(),v))))
     }}
 }
+
+#[test]
```

Comment:
Do the tests need to be inside a module guarded by `#[cfg(test)]`?

Reply:
Done